### ISSUE TITLE
false positive 'selector-nest-combinators'

### ISSUE THREAD

**Issue by Cole-Beckwith-SP** [kristerkari/stylelint-scss#306]
Given a value 'always' and a stylesheet:

```
.class-name {
	#{if(&, "&", "")} {

	}	
}
```

The linter will report the error:

`'Expected combinator "}" of type "tag" to be in a nested form'`

**Comment by kristerkari** [comment]
Thanks! Looks like the rule needs to ignore interpolation.

**Comment by Cole-Beckwith-SP** [comment]
Thanks for this project!

**Comment by kristerkari** [comment]
Thanks for reporting the issue :) 

I already have fix for the issue, so I'll try to release a patch tonight or tomorrow.

**Comment by kristerkari** [comment]
fix released in v3.5.2
https://github.com/kristerkari/stylelint-scss/releases/tag/3.5.2
